templates/src/index.js

```diff
@@ -0,0 +1,25 @@
+import { getOptions } from 'loader-utils';
+import validateOptions from 'schema-utils';
+import schema from './options.json';
+
+export const raw = true;
+
+export default function loader(source) {
+  const { version, webpack } = this;
```

Comment:
This will fail the current  `ValidationError` test in `Errors.test.js`, we could either upgrade that test to use `webpack` (`helpers/compiler.js`)  instead of `loader.call()` or move the code beyond `validateOptions` (L12+)

Reply:
Adds a validation error test that uses the compiler. It's probably overkill to run schema validation through Webpack but given the number of times we have been bitten by some unit test that works fine until you use the compiler, i'm fine with adding a few extra seconds to a tests execution time.

Reply:
Yep, 💯 in favor of handling it that way. `loader.call` should be avoided whenever possible, it's a bad pseudo testing pattern in general and the code structure of a `loader/plugin` should not be influenced by a unit test at all :)